src/components/dialogs/pick-entity-or-conversion.vue

```diff
@@ -134,18 +171,14 @@ export default class PickEntityOrConversionDialog extends CommonMixin {
     }
   }
 
-  get entity_type_cd (): EntityType {
+  get entity_type_cd (): EntityTypes {
     return this.getEntityTypeCd
   }
 
-  set entity_type_cd (value: EntityType) {
+  set entity_type_cd (value: EntityTypes) {
     this.setEntityTypeCd(value)
   }
 
-  get locationText (): string {
-    return (this.getLocationText === 'BC') ? 'British Columbia' : this.getLocationText
-  }
```

Comment:
It turns out the store getter `getLocationText` already returns "British Columbia", so this computed is not needed.